.eslintrc.js

```diff
@@ -137,6 +137,12 @@ module.exports = {
         '@typescript-eslint/explicit-function-return-type': 0,
         '@typescript-eslint/unbound-method': 0,
 
+        'jest/expect-expect': [
+          'error',
+          {
+            assertFunctionNames: ['expect', 'validate'],
```

Comment:
Why this?

Reply:
Otherwise eslint trows an error, it expects `expect` keywork inside each unit test by default

Reply:
Yes, that should be enforced, please only mark required tests and don't disable this globally.

Reply:
Yeah, I did not disable anything. I only extended allowed keywords, now it is no only `expect`, but `validate` as well

Reply:
So with this change, either `expect` or `validate` are allowed *globally*? @viceice is that a concern if so? @pret-a-porter alternatively is it too hard to set this value just where applicable?

Reply:
Yes, now expect or validate are allowed globally. I am not sure, that possible to apply this eslint rule only for specific folder.

Reply:
@viceice does this sound ok with you?

Reply:
Done